connection.js

```diff
@@ -179,6 +182,21 @@ function connect(id, password, roomName) {
             APP.store.dispatch(setPrejoinDisplayNameRequired());
         }
 
+        /**
+         * Logs jwt validation errors from xmpp and from the client-side validator.
+         *
+         * @param {string} message -The error message from xmpp.
+         * @returns {void}
+         */
+        function _logJwtErrors(message) {
```

Comment:
Some of the validations here apply only to JaaS users, please let's make that clear so other Jitsi users whi authenticate with a JWT don't see unexpected errors.

Reply:
aud, iss and sub ~ kid validations are now JaaS only.